Meta/Websites/serenityos.org/index.html

```diff
@@ -22,6 +22,7 @@ <h1>SerenityOS</h1>
     <li><a href="https://github.com/SerenityOS/serenity">SerenityOS on GitHub</a></li>
     <li><a href="https://discord.com/invite/29gCcKsXkF">SerenityOS Discord Server</a> <font color=red>(join here to chat!)</font></li>
     <li><a href="faq/">Frequently asked questions</a></li>
+    <li><a href="https://man.serenityos.org">SerenityOS man pages</a></li>
```

Comment:
Note that this directory is not directly deployed to serenityos.org. In the past, Andreas used to make adjustments on his local copy and push the changes to the main repo occasionally. Therefore I don't know whether this state is even current anymore; pinging @awesomekling .

Reply:
I think I'll remove this commit from the PR then. Hopefully Andreas will add it to the homepage 😃